src/numeric_index.h

```diff
@@ -52,6 +52,11 @@ typedef struct rtNode {
   NumericRange *range;
 } NumericRangeNode;
 
+typedef struct {
+  size_t sz;
```

Comment:
This probably should be reduced in order to fit into 8 bytes; let's make the stack work less!